### PR TITLE
Program to check a string a pangram or not issue#925

### PR DESCRIPTION
**Pull request by samya02**
issue #925 
I have added the code in Code/Python check pangram.
Please review it!
Thanks!!

### CODE REVIEW

**Comment by samya02** [comment]
Kindly review my PR @raksha009 .
Thanks!

**Comment by raksha009** [comment]
Plz link yr code with respective folder :). Go through the guidelines and the pinned messages

**Comment by samya02** [comment]
@raksha009 I have linked the code. Kindly review !
Thanks:)

File: Algorithm/Number_theory/readme.md [hunk 1]
```diff
@@ -39,3 +39,4 @@
 # Questions related to Number Theory
 
 - GCD of two number using Euclidean Algorithm ----> [Java](/Code/Java/GCD.java)
+- Program to check a string a pangram or not ----> [Python](https://github.com/Algo-Phantoms/Algo-Tree/blob/071d6ba576ef9f2140aa913920b1c4c6daa89689/Code/Python/check%20pangram.py)
```
Comment (raksha009) [inline Algorithm/Number_theory/readme.md:42]:
```suggestion
- Program to check a string a pangram or not ----> [Python](/Code/Python/check_pangram.py) 
```

also, change yr file name to check_panagram.py

File: Code/Python/check pangram.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#Program to check if a given string is pangram or not
```
Comment (raksha009) [inline Code/Python/check pangram.py:1]: write description in multiline comments

File: Code/Python/check pangram.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        
+        if alpha not in string.lower():   
+            return False
+    
+    
+    return True
+
+
+#main
+string=input("Enter any string: ")
```
Comment (raksha009) [inline Code/Python/check pangram.py:26]:
```suggestion
string = input("Enter any string: ")
```
Add space around the operators like <, > . =. == etc

File: Code/Python/check pangram.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+if(pangram(string)==True):
+    
+    print("\n The given string is a pangram")
+          
+else:
+    print("\n The given string is not a pangram")
+
+
+    '''Test cases: 
+1) I/P:The Quick Brown Fox Jumps Over The Lazy Dog
```
Comment (raksha009) [inline Code/Python/check pangram.py:39]: Follow 
'''
Test Cases
Input : 
Output :

File: Code/Python/check pangram.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+   O/P:The given string is a pangram
+
+2) I/P:2brown5! (including special characters and digits)
+   O/P:The given string is not a pangram
+'''
+
+#Time complexity:O(n)
```
Comment (raksha009) [inline Code/Python/check pangram.py:46]: 1. Include this in multline comment
2. Write space complexity too
3. specify n

**Review by raksha009 — CHANGES_REQUESTED** [review]

File: Code/Python/check pangram.py (hunk 5 above)
Comment (samya02) [reply Code/Python/check pangram.py]: done!

File: Code/Python/check pangram.py (hunk 4 above)
Comment (samya02) [reply Code/Python/check pangram.py]: done!

File: Code/Python/check pangram.py (hunk 3 above)
Comment (samya02) [reply Code/Python/check pangram.py]: done!

File: Code/Python/check pangram.py (hunk 2 above)
Comment (samya02) [reply Code/Python/check pangram.py]: made the changes! :)

File: Algorithm/Number_theory/readme.md (hunk 1 above)
Comment (samya02) [reply Algorithm/Number_theory/readme.md]: done!

**Comment by samya02** [comment]
@raksha009 Made all the required changes! Please review!
Thank You!

**Review by raksha009 — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**